---a/net/minecraft/entity/ai/EntityAIWatchClosest2.java
+++b/net/minecraft/entity/ai/EntityAIWatchClosest2.java
@@ -2,14 +2,13 @@
 
 import net.minecraft.entity.EntityLiving;
 
-public class EntityAIWatchClosest2 extends EntityAIWatchClosest
-{
+public class EntityAIWatchClosest2 extends EntityAIWatchClosest {
+
     private static final String __OBFID = "CL_00001590";
 
-    public EntityAIWatchClosest2(EntityLiving p_i1629_1_, Class p_i1629_2_, float p_i1629_3_, float p_i1629_4_)
-    {
+    public EntityAIWatchClosest2(EntityLiving p_i1629_1_, Class p_i1629_2_, float p_i1629_3_, float p_i1629_4_) {
         super(p_i1629_1_, p_i1629_2_, p_i1629_3_, p_i1629_4_);
         this.setMutexBits(3);
     }
-}
+}
